packages/react/src/patterns/sections/LeadSpace/LeadSpace.js

```diff
@@ -25,17 +21,62 @@ const { prefix } = settings;
  *
  * @param {string} variation variation of the pattern
  * @param {string} theme theme of the pattern
+ * @param {string} type switches between centered or default
+ * @param {object} image object
  * @returns {string} classnames
  */
-const className = (variation, theme) =>
+const className = (variation, theme, type, image) =>
   classnames(
     `${prefix}--leadspace`,
     theme && `${prefix}--leadspace--${theme}`,
     {
-      [`${prefix}--leadspace--productive`]: variation === 'productive',
+      [`${prefix}--leadspace--productive`]: type === 'small',
+    },
+    {
+      [`${prefix}--leadspace--centered${theme && '--g100'}`]:
+        type === 'centered',
+    },
+    {
+      [`${prefix}--leadspace--centered__image`]: image && type === 'centered',
     }
   );
 
+/**
+ * @param {string} type returns centered or default
+ * @param {string} element returns element name
+ * @returns {string} classnames
+ */
+function centeredClassname(type, element) {
+  if (type === 'centered') {
+    return `${prefix}--leadspace--centered__${element}`;
+  } else return `${prefix}--leadspace__${element}`;
+}
+
+/**
+ *
+ * @param {string} type type
+ * @param {object} image image
+ * @returns {object} returns either image component or the centered image div
+ */
+function imageClassname(type, image) {
+  if (type === 'centered') {
+    return (
+      <div
+        data-autoid={`${stablePrefix}--leadspace--centered--mobile__image`}
+        className={`${prefix}--leadspace--centered--mobile__image`}>
+        <img src={image.default} alt={image.alt} />
+      </div>
+    );
+  } else
+    return (
+      <LeadSpaceImage
```

Comment:
We can actually use the `Image` component here now instead and remove `LeadSpaceImage`